Tests/Security/Authorization/Interception/MethodSecurityInterceptorTest.php

```diff
@@ -270,7 +270,10 @@ protected function getInterceptor(MetadataFactoryInterface $metadataFactory = nu
             ;
         }
 
-        $securityContext = $this->getMockBuilder('Symfony\Component\Security\Core\SecurityContext')
+        $securityContextClass = class_exists('Symfony\Component\Security\Core\SecurityContext')
+            ? 'Symfony\Component\Security\Core\SecurityContext'
+            : 'Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface';
```

Comment:
the check should be reverted, to prefer mocking the new API IMO. Or even better having 2 tests:

- a test using the new API (and skipped for Symfony 2.5 and older where it does not exist)
- a test marked as ``@legacy`` (to allow deprecation notices) testing the old API (and skipped on Symfony 3 where the legacy interface does not exist anymore)

This would allow to test the BC layer too.